cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -99,15 +99,15 @@
 
             {% block primary_nav %}
                 {% if flag_enabled('MEGA_MENU_BACKEND_V2') %}
-                    {{ mega_menu() }}
+                    {{ mega_menu( language=language ) }}
```

Comment:
```suggestion
                    {{ mega_menu() }}
```

This change is not needed because in the new backend, `mega_menu` here is not calling the template, but is calling a Jinja function is defined as a [`contextfunction`](https://github.com/cfpb/cfgov-refresh/blob/a07fceea277b766effa3b16a497a2a933ab3943c/cfgov/mega_menu/jinja2tags.py#L53) which automatically gets passed all context variables, including `language`.